gulpfile.js

```diff
@@ -0,0 +1,88 @@
+var gulp = require('gulp'),
+	phpcs = require('gulp-phpcs'),
+	phpunit = require('gulp-phpunit'),
+	sourcemaps = require('gulp-sourcemaps'),
+	lesshint = require('gulp-lesshint'),
+	less = require('gulp-less'),
+	cssmin = require('gulp-cssmin'),
+	LessAutoprefix = require('less-plugin-autoprefix'),
+	autoprefix = new LessAutoprefix({
+		cascade: false,
+		browsers: ['> 1%', 'ie 8-11', 'Firefox ESR'],
+	}),
+	rename = require('gulp-rename'),
+	livereload = require('gulp-livereload');
+
+gulp.task('phpcs', function() {
+	return (
+		gulp
+			.src(['./**/*.php', '!vendor/**/*.*', '!node_modules/**/*.*'])
+			// Validate files using PHP Code Sniffer
+			.pipe(
+				phpcs({
+					bin: 'vendor/bin/phpcs',
+					standard: 'phpcs.ruleset.xml',
+				})
+			)
+			// Log all problems that was found
+			.pipe(phpcs.reporter('log'))
```

Comment:
I don't actually get a list of errors when running `gulp phpcs` that I get when I run just `phpcs`. It doesn't appear to be outputting anything. I even tried the file output and got no result.